### PR TITLE
jwt: add page

### PR DESCRIPTION
**Pull request by ImShakthi**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,9 +4,11 @@
   "description": "Simplified, community-driven man pages",
   "dependencies": {
     "glob": "7.1.3",
+    "husky": "^2.2.0",
```
Comment (agnivade) [inline package.json]: These are unrelated changes. Let us revert them

File: pages/common/jwt.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# jwt
+
+> `jwt-cli` is a command line tool to help you work with JSON Web Tokens (JWTs).
```
Comment (agnivade) [inline pages/common/jwt.md]: Remove "`jwt-cli` is". Also "help you"

File: pages/common/jwt.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+> A super fast CLI tool to decode and encode JWTs built in Rust.
```
Comment (agnivade) [inline pages/common/jwt.md]: I think this line is redundant. We can remove this.

File: pages/common/jwt.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+> Encryption algorithms available are HS256,HS384, HS512, RS256, RS384, RS512, ES256, ES384.
+> For more information: <https://github.com/mike-engel/jwt-cli>.
+
+- Decode a JWT:
+
+`jwt decode <JWT string>`
```
Comment (agnivade) [inline pages/common/jwt.md]: Please change all the tokens to `{{}}` format and use snake_case.

File: pages/common/jwt.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+- Decode a JWT as JSON:
+
+`jwt decode -j <JWT string>`
+
+- Encode a JSON to JWT:
```
Comment (agnivade) [inline pages/common/jwt.md]: Encode a JSON string to a JWT

File: pages/common/jwt.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+> `jwt-cli` is a command line tool to help you work with JSON Web Tokens (JWTs).
+> A super fast CLI tool to decode and encode JWTs built in Rust.
+> Encryption algorithms available are HS256,HS384, HS512, RS256, RS384, RS512, ES256, ES384.
+> For more information: <https://github.com/mike-engel/jwt-cli>.
+
+- Decode a JWT:
+
+`jwt decode <JWT string>`
+
+- Decode a JWT as JSON:
```
Comment (agnivade) [inline pages/common/jwt.md]: Decode a JWT as a JSON string

File: pages/common/jwt.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+`jwt decode -j <JWT string>`
+
+- Encode a JSON to JWT:
+
+`jwt encode --alg <ALGO> --secret <SECRET> '<JSON>'`
```
Comment (agnivade) [inline pages/common/jwt.md]: Also, give concrete values for algo from the sub-heading

File: pages/common/jwt.md (hunk 3 above)
Comment (schneiderl) [reply pages/common/jwt.md]: Also, we tend to omit adjectives ("complete", "open source", "super fast") in the main description. Stick to describing what it is and what it does :)

**Comment by ImShakthi** [comment]
@agnivade Thank you for the review, it all make senses. I will update it based on your comments.

**Comment by ImShakthi** [comment]
@agnivade I have incorporated your feedback. :)

File: pages/common/jwt.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+`jwt decode {{jwt_string}}`
+
+- Decode a JWT as a JSON string:
+
+`jwt decode -j {{jwt_string}}`
+
+- Encode a JSON string to a JWT:
+
+`jwt encode --alg HS256 --secret 1234567890 '{{json_string}}'`
```
Comment (agnivade) [inline pages/common/jwt.md]: The algs and secrets should still be inside tokens.

Comment (sbrl) [reply pages/common/jwt.md]: i.e. `{{` and `}}`.

Comment (sbrl) [reply pages/common/jwt.md]: i.e. `{{` and `}}`.

Comment (ImShakthi) [reply pages/common/jwt.md]: @agnivade and @sbrl I have incorporated those changes :)
